From b9ad10c075441183e22cb9724762d4a1174585a6 Mon Sep 17 00:00:00 2001
From: Sean Niesen <seanniesen@gmail.com>
Date: Wed, 25 Feb 2015 23:32:52 -0800
Subject: [PATCH 05/25] format date correctly

---
 app.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/app.js b/app.js
index 92a1b0b..b50cbda 100644
--- a/app.js
+++ b/app.js
@@ -24,12 +24,13 @@ app.controller('MainCtrl', [
 	   			while (startDateMilli <= endDateMilli) {
 
 	   				var currentDate = new Date(startDateMilli);
+	   				var splitDate = currentDate.toString().split(" ").slice(0, 4).join(" ");
 	 					var sunriseurl = "http://api.sunrise-sunset.org/json?lat=" + results[0].geometry.location.lng() + "&lng=" + results[0].geometry.location.lat() + "&date=" + currentDate + "&callback=mycallback";
 			   		var response = $.ajax({
 											    		url: sunriseurl,
 											    		dataType: "JSONP",
 											    		success: function(data) {
-											    			$("#sunrises").append("<tr class='day day" + i + "'><td>" + currentDate + "</td><td>" + data.results.sunrise + "</td></tr>");
+											    			$("#sunrises").append("<tr class='day day" + i + "'><td>" + splitDate + "</td><td>" + data.results.sunrise + "</td></tr>");
 											    			// $scope.sunrise = data.results.sunrise;
 											    			times.push("arf");
 										    			i += 1;
-- 
2.1.3.36.g8e36a6d

